sys/vane/clay.hoon

```diff
@@ -195,6 +195,8 @@
       mon/(map term beam)                               ::  mount points
       hez/(unit duct)                                   ::  sync duct
       cez/(map @ta crew)                                ::  permission groups
+      cue/(qeu [duct task:able])                        ::  queued requests
+      tip/@da                                           ::  last write date
```

Comment:
These should maybe get comments above along with the rest of the `raft`. There's room up there for a short explanation of what this gets used for/solves.

Reply:
Ok, I'll add some more docs and then merge this.